### PR TITLE
Edited concepts.tex

### PR DESCRIPTION
**Pull request by amazein**
Changed to TTTT

### CODE REVIEW

**Comment by cannin** [comment]
I think there are issues with the changes.

File: Specification/sources/concepts.tex [hunk 1]
```diff
@@ -4,7 +4,7 @@ \chapter{Language Overview}
...
+ TTTT TEXT briefly describe what SBGN \PDl is about, let's give a brief overview of some of the relevant concepts with the help of an example shown in \fig{eg1}. It is a simple map for part of a mitogen-activated protein kinase (MAPK) cascade.  The larger nodes in the figure (some of which are in the shape of rounded rectangles and others in the shape of circles) represent biological materials---things like macromolecules and simple chemicals (NB: the nodes representing physical entities (or proxies to physical entities) will always be colored in yellow in this document. Color is not part of the SBGN specification though).
```
Comment (cannin) [inline Specification/sources/concepts.tex]: This should be fixed.

Comment (amazein) [inline Specification/sources/concepts.tex]: Replying

**Comment by cannin** [comment]
Test pull request ignore.

**Pull request closed without merging**